src/components/App.css

```diff
@@ -37,3 +37,11 @@
   margin-top: 25px;
   margin-right: 20px;
 }
+
+/* Utils */
+.vertical-center {
+  top: 50%;
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);
```

Comment:
we can drop the prefixes because we have autoprefixer on:

this is the output w/o it
```css
/* Utils */
.vertical-center {
  top: 50%;
  -webkit-transform: translateY(-50%);
          transform: translateY(-50%);
}
```